tests/test_loader.py

```diff
@@ -528,7 +538,7 @@ def test_get_output_value_list(self):
         self.assertEqual(il.get_output_value('name'), ['foo', 'bar'])
         loaded_item = il.load_item()
         self.assertIsInstance(loaded_item, self.item_class)
-        self.assertEqual(loaded_item, dict({'name': ['foo', 'bar']}))
+        self.assertEqual(self.to_dict(loaded_item), dict({'name': ['foo', 'bar']}))
```

Comment:
Is self.to_dict needed because item loaders can return dataclass instances now? If so, could you please add tests for this behavior?

Reply:
Also, could you please check Item Loaders docs, and update it accordingly? I think it still says that Item instances are expected, or dict-like objects, which is no longer true.

Reply:
Most testing methods The `InitializationTestMixin` have an assertion to check if the produced item matches the expected type: `self.assertIsInstance(loaded_item, self.item_class)`. Would that be enough?

Reply:
Aha, I missed that. Yes, I think this is enough.